lib/scripts/services/contextMenu.js

```diff
@@ -124,30 +124,17 @@ export class ContextMenu {
       }
     }
 
-    if (isTranslatedWord) {
-
-      // Hash storage format:
-      // format: {targetLanguage: {word1: E/N/H, word2: E/N/H}}
-      // E -> easy, N -> normal, H -> hard
+    if (isTranslatedWord){
+        // Hash storage format:
+        // format: {word1: E/N/H, word2: E/N/H}
```

Comment:
Why did you decide to stop having separate buckets for different target languages?  (I am just curious about your reasons)